x-pack/plugins/enterprise_search/server/lib/enterprise_search_config_api.test.ts

```diff
@@ -38,51 +38,63 @@ describe('callEnterpriseSearchConfigAPI', () => {
       external_url: 'http://some.vanity.url/',
       read_only_mode: false,
       ilm_enabled: true,
+      is_federated_auth: false,
       configured_limits: {
-        max_document_byte_size: 102400,
-        max_engines_per_meta_engine: 15,
+        app_search: {
+          engine: {
+            document_size_in_bytes: 102400,
+            source_engines_per_meta_engine: 15,
+          },
+        },
+        workplace_search: {
+          custom_api_source: {
+            document_size_in_bytes: 102400,
+            total_fields: 64,
+          },
+        },
+      },
+    },
+    current_user: {
+      name: 'someuser',
+      access: {
+        app_search: true,
+        workplace_search: false,
       },
       app_search: {
-        account_id: 'some-id-string',
-        onboarding_complete: true,
+        account: {
+          id: 'some-id-string',
+          onboarding_complete: true,
+        },
+        role: {
+          id: 'account_id:somestring|user_oid:somestring',
+          roleType: 'owner',
```

Comment:
Note to self - I'm checking with Brian to see if this camelCased (vs snake_cased) var was a mistake or if it's an forced ent-search formatting thing. Possible it might change, also possible it might stay the same 🤷